plugin/src/main/scala/scoverage/CoverageFilter.scala

```diff
@@ -28,8 +28,13 @@ object AllCoverageFilter extends CoverageFilter {
 class RegexCoverageFilter(
     excludedPackages: Seq[String],
     excludedFiles: Seq[String],
-    excludedSymbols: Seq[String]
+    excludedSymbols: Seq[String],
+    reporter: scala.tools.nsc.reporters.Reporter
+
 ) extends CoverageFilter {
+  reporter.echo(s"scoverage excludedPackages: ${excludedPackages}")
+  reporter.echo(s"scoverage excludedFiles: ${excludedFiles}")
+  reporter.echo(s"scoverage excludedSymbols: ${excludedSymbols}")
```

Comment:
Adding some logging (for Scala2).

Reply:
What do you think about only displaying these if there are being used? For example if I only exclude a package, then that's the only message I see, `scoverage excludedPackages: foo`?

Reply:
Hi Chris. Was thinking to always display it to make sure that you can also see that it is not set. But happy to test for it to it to be not empty, if you think that it better.

Reply:
I think it sort of delivers the same message. If it doesn't appear, then it's not being picked up. Since we didn't have this in here before, let's limit the noise and only display them if they are set.